airbyte-webapp/src/packages/cloud/views/settings/integrations/DbtCloudSettingsView.tsx

```diff
@@ -12,21 +12,39 @@ import styles from "./DbtCloudSettingsView.module.scss";
 
 export const DbtCloudSettingsView: React.FC = () => {
   const { mutate: submitDbtCloudIntegrationConfig, isLoading } = useSubmitDbtCloudIntegrationConfig();
+  const [hasValidationError, setHasValidationError] = useState<boolean>(false);
+  const [validationErrorMessage, setValidationErrorMessage] = useState<string | null>(null);
```

Comment:
Since the empty string is falsey, I ended up initializing it with that, letting the type be inferred as a plain, unconditional `string` while keeping the same empty message semantics.